### PR TITLE
Upgrade shader code

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
Upgrades shader code to use OpenGL `#version 300 es`. The desktop volume viewer uses `#version 330 core` but that is not available in WebGl. The closest version seems to be 300 es.

### CODE REVIEW

**Review by mcmcgrath13 — COMMENTED** [review]
I defer to the graphics experts on this one

File: src/Aframe/vertex-shader.vert [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+
+in vec3 position;   // Position vector of the vertex (WebGL)
+out vec3 vUV;       // Coordinates of the texture
+out vec3 camPos;    // Coordinates of the camera
+
 uniform float zScale;
-varying  vec3 vUV;      // 3D texture coordinates for texture lookup in the fragment shader
-varying  vec3 camPos;   // Position of the camera
+uniform mat4 modelViewMatrix;   // Position of model in world space (WebGL)
+uniform mat4 projectionMatrix;  // Position of the model in clip space (WebGL)
```
[the remark this answers is not in the document]
Comment (RobertGemmaJr) [reply src/Aframe/vertex-shader.vert]: Oh okay that totally makes sense! I updated the comments

File: src/Aframe/vertex-shader.vert [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+
+in vec3 position;   // Position vector of the vertex (WebGL)
+out vec3 vUV;       // Coordinates of the texture
+out vec3 camPos;    // Coordinates of the camera
+
 uniform float zScale;
-varying  vec3 vUV;      // 3D texture coordinates for texture lookup in the fragment shader
-varying  vec3 camPos;   // Position of the camera
+uniform mat4 modelViewMatrix;   // world -> camera space transformation matrix (WebGL) 
+uniform mat4 projectionMatrix;  // camera -> clip space transformation matrix (WebGL)
```
Comment (kmilo9999) [inline src/Aframe/vertex-shader.vert:9]: I thought Webgl (threejs )  already [defined this matrices](https://threejs.org/docs/#api/en/renderers/webgl/WebGLProgram) . Do you need to redefine them to go up on glsl ?

Comment (RobertGemmaJr) [reply src/Aframe/vertex-shader.vert:9]: It defines them with `THREE.ShaderMaterial` but I'm using `THREE.RawShaderMaterial`. My linter was getting very mad when I was using `ShaderMaterial` so I decided to switch. THREE still handles all the setting/updating of the matrices

**Review by kmilo9999 — APPROVED** [review]
It looks good!

**Pull request merged**